fe1-web/model/network/method/message/__tests__/Message.test.ts

```diff
@@ -29,7 +29,7 @@ const mockLao = Lao.fromState(laoState);
 
 const mockPublicKey = testKeyPair.publicKey;
 const mockPrivateKey = testKeyPair.privateKey;
-const mockPopToken = PopToken.fromState({
+export const mockPopToken = PopToken.fromState({
```

Comment:
If you want to make it a constant, I'd rather put it in a separate file. It feels weird to me to have a constant in this file that is exported somewhere else.